modules/v1/workspaces/resources/FolderResource.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+
+namespace app\modules\v1\workspaces\resources;
+
+
+use app\modules\v1\users\resources\UserResource;
+use app\modules\v1\workspaces\models\Folder;
+use Yii;
+use yii\db\ActiveQuery;
+
+/**
+ * Class FolderResource
+ *
+ * @package app\modules\v1\workspaces\resources
+ */
+class FolderResource extends Folder
+{
+    const VIDEO = 'video';
+
+    public function fields()
+    {
+        return [
+            'id',
+            'parent_id',
+            'workspace_id',
+            'name',
+            'label',
+            'depth',
+            'size',
+            'is_file',
+            'is_timeline_folder',
+            'file_path',
+            'mime' => function () {
+                $mime = explode('/', $this->mime)[0];
```

Comment:
@skavt It is better to move such logic to `Folder` model and in here we'll have such code: `return $this->getMime();`